gradle.properties

```diff
@@ -1,32 +1,33 @@
 org.gradle.caching=true
 group=io.ballerina.lib
-version=1.3.0-SNAPSHOT
+version=1.4.0-SNAPSHOT
 
-ballerinaLangVersion=2201.11.0-20241117-133400-a3054b77
+ballerinaLangVersion=2201.12.6
```

Comment:
Do we have to limit this to the patch version ? Can't we release on U12 itself ?

Reply:
Fixed with [7524c23](https://github.com/ballerina-platform/module-ballerinax-ibm.ibmmq/pull/65/commits/7524c2334f8a68fac83e7a30a0039b475f805fec)